static/app/components/globalDrawer/useDrawerResizing.tsx

```diff
@@ -65,11 +69,24 @@ export function useDrawerResizing({
     );
 
   useLayoutEffect(() => {
-    panelRef.current?.style.setProperty('--drawer-width', `${persistedWidthPercent}%`);
-  }, [persistedWidthPercent]);
+    if (isSmallScreen) {
```

Comment:
is this still needed after the last change?

Reply:
I believe so